06-hot-reloading-with-directives/services/accounts/schema.graphql

```diff
@@ -0,0 +1,13 @@
+type User @key(selectionSet: "{ id }") {
+  id: ID!
+  name: String
+  username: String
+}
+
+scalar _Key
+
+type Query {
+  me: User
+  _users(keys: [_Key!]!): [User] @merge
```

Comment:
What is the shape of ` _Key`... is this a federation entity service?

Reply:
There is a later example where key is typed. It has the shape of the requested selection sets by default, but you can modify that with the key arg

Reply:
I basically tried to give both examples here.